app/models/embeddable_media.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
```

Comment:
This was an auto add after database migrations. Can be removed from branch, if needed.